{{cookiecutter.github_repository}}/.envs/.dev/.env.sample

```diff
@@ -0,0 +1,77 @@
+# General
+# ------------------------------------------------------------------------------
+USE_DOCKER=yes
+IPYTHONDIR=/app/.ipython
+
+# Redis
+# ------------------------------------------------------------------------------
+REDIS_URL=redis://redis:6379/0
+
+# Celery
+# ------------------------------------------------------------------------------
+
+# Flower
+CELERY_FLOWER_USER=dev
+CELERY_FLOWER_PASSWORD="{{ cookiecutter.main_module|title|truncate(3, True, '') }}1234!"
+
+
+# Databases
+# ==============================
+# DATABASE_URL=postgres://username:password@yourhost.com:5432/database_name
```

Comment:
We could remove `DATABASE_URL`.

Reply:
Oh, this is a sample env file, so sample values here are okay. This is to guide people to replace them by actual values when they place an actual `.env` file.